vue.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require("path");
 
 module.exports = {
-  chainWebpack: (config) => {
+  chainWebpack: function (config) {
```

Comment:
Why did you replace arrow function?

Reply:
Probably Łukasz was not  familiar with ES6 syntax (but that's just my guess), I'm gonna revert the change in a minute.

Reply:
I didn't even know we changed something like that. I'm gonna change back to arrow function.